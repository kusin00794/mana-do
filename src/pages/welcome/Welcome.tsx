import React from 'react';

const Welcome = () => {
  return (
    <>
      <div className="row">
        <div className="col-12">
          <h1 className="text-center">Welcome to To Do App</h1>
        </div>
      </div>
    </>
  );
};

export default Welcome;
