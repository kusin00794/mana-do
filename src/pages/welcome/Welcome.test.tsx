import React from 'react';
import { BrowserRouter } from 'react-router-dom';

import { testSnapshots } from '../../utils/test';
import Welcome from './Welcome';

const Wrapper = (props: any) => (
  <BrowserRouter>
    <Welcome {...props} />
  </BrowserRouter>
);
describe('Welcome Page', () => {
  testSnapshots(Wrapper, [
    {
      props: {},
      description: 'render Welcome Page',
    },
  ]);
});
