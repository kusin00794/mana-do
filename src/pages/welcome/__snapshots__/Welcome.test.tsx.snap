// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Welcome Page snapshots render Welcome Page 1`] = `
<div
  className="row"
>
  <div
    className="col-12"
  >
    <h1
      className="text-center"
    >
      Welcome to To Do App
    </h1>
  </div>
</div>
`;
